presto-orc/src/test/java/com/facebook/presto/orc/OrcTester.java

```diff
@@ -686,6 +697,19 @@ private OrcReaderSettings(Map<Integer, Map<Subfield, TupleDomainFilter>> columnF
             return requiredSubfields;
         }
 
+        public boolean hasExpectedError()
```

Comment:
One method is good enough: `public Optional<String> getExpectedErrorMessage()`

Reply:
Okay